config/github-webhook/verify-signed-off.rb

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env ruby
+
+#
+# Simple(ish) Ruby Sinatra bot to verify that all commits on a Github
+# pull request have a Signed-off-by line in the commit message.
+#
+# Note that this code was heavily inspired by
+# http://git-scm.com/book/en/v2/GitHub-Scripting-GitHub
+#
+##################################################################################
+
+# Yum-installed gems; apparently CentOS doesn't automatically add them
+# to the ruby search path.  :-( There's probably a better way to do
+# this, but I'm not One with the Ruby...
+$LOAD_PATH.unshift '/usr/lib/ruby/gems/1.8/gems/crack-0.1.8/lib'
```

Comment:
What version of ruby is running on `flatbed`?  If it's from the EOL-ed v1.8 series (or worse, earlier), then I think these will get fixed with a `require 'rubygems'` line instead of all of these `$LOAD_PATH.unshift` statements.